pkg/injector/webhook.go

```diff
@@ -32,21 +31,23 @@ var (
 	codecs       = serializer.NewCodecFactory(runtime.NewScheme())
 	deserializer = codecs.UniversalDeserializer()
 
-	kubeSystemNamespaces = []string{
-		metav1.NamespaceSystem,
-		metav1.NamespacePublic,
+	kubeSystemNamespaces = map[string]interface{}{
+		metav1.NamespaceSystem: nil,
```

Comment:
What was the goal of converting this from a slice to a map?

Reply:
The goal was probably to optimize lookup in L232, which doesn't add significant value given the number of kube namespaces to iterate through is just 2.

For simplicity, I'd revert this particular change.

Reply:
Simplified this further.